README.md

```diff
@@ -1,37 +1,81 @@
 
-## Installing ONLYOFFICE Document Server
+Supported formats:
 
-You will need an instance of ONLYOFFICE Document Server that is resolvable and connectable both from Nuxeo and any end clients. If that is not the case, use the official ONLYOFFICE Document Server documetnation page: [Document Server for Linux](http://helpcenter.onlyoffice.com/server/linux/document/linux-installation.aspx). ONLYOFFICE Document Server must also be able to POST to Nuxeo directly.
+* For viewing and editing: DOCX, XLSX, PPTX.
+* For viewing only: PDF, ODT, ODS, ODP, DOC, XLS, PPT.
 
-The easiest way to start an instance of ONLYOFFICE Document Server is to use [Docker](https://github.com/onlyoffice/Docker-DocumentServer).
+## Installing ONLYOFFICE Document Server
 
+You will need an instance of ONLYOFFICE Document Server that is resolvable and connectable both from Nuxeo and any end-clients. ONLYOFFICE Document Server must also be able to POST to Nuxeo directly.
 
-## Compiling Nuxeo ONLYOFFICE integration plugin
+You can install free Community version of ONLYOFFICE Document Server or scalable enterprise-level Integration Edition.
 
+To install free Community version, use [Docker](https://github.com/onlyoffice/Docker-DocumentServer) (recommended) or follow [these instructions](https://helpcenter.onlyoffice.com/server/linux/document/linux-installation.aspx) for Debian, Ubuntu, or derivatives.  
 
+To install Integration Edition, follow instructions [here](https://helpcenter.onlyoffice.com/server/integration-edition/index.aspx).
 
+Community Edition vs Integration Edition comparison can be found [here](#onlyoffice-document-server-editions).
 
 ## Installing Nuxeo ONLYOFFICE integration plugin
 
+Install it from [marketplace](https://connect.nuxeo.com/nuxeo/site/marketplace).
 
+You can also install it using [nuxeoctl](https://doc.nuxeo.com/nxdoc/installing-a-new-package-on-your-instance/).
+```
+nuxeoctl mp-install /path/to/onlyoffice-nuxeo-package-x.x.zip
+```
 
-## Configuring Nuxeo CONLYOFFICE integration plugin
+## Configuring Nuxeo ONLYOFFICE integration plugin
 
+Edit [nuxeo.conf](https://doc.nuxeo.com/nxdoc/configuration-parameters-index-nuxeoconf/) and add following lines:
+```
+onlyoffice.docserv.url=http://documentserver/
+onlyoffice.jwt.secret=
+```
+If you used Docker to install ONLYOFFICE Document Server, use information from [this repo](https://github.com/ONLYOFFICE/Docker-DocumentServer/#available-configuration-parameters) to configure JWT.
 
+If you used other installation options, check the [API documentation](https://api.onlyoffice.com/editors/signature/) for configuring JWT on the Document Server side.  
 
-## How it works
+## Compiling Nuxeo ONLYOFFICE plugin
 
-The ONLYOFFICE integration follows the API documented [here](https://api.onlyoffice.com/editors/basic):
+To build Nuxeo plugin, the following steps must be performed for Ubuntu 14.04:
```

Comment:
Ubuntu 14.04 eol soon
Same with java 8

I don't think those steps depend of those old versions, maybe update it?

Reply:
java 8 is a minimum supported version for compiling a plugin, you can compile under java 11 as well, we can change the instruction if you wish

we can omit the ubuntu version because it is not really relevant here or we can bump it, which do you prefer?

Reply:
> java 8 is a minimum supported version for compiling a plugin, you can compile under java 11 as well, we can change the instruction if you wish

I think it's enought to change lines to 
`use the following commands to install Open JDK (version 8 or any newer versions)`

> we can omit the ubuntu version because it is not really relevant here 

I think just remove note about 14.04 like:
`To build Nuxeo plugin, the following steps must be performed for Ubuntu:`

Reply:
actually LTS version of nuxeo runs on java 8, so while we can compile it under java 11 it might not work properly, so better leave java version as is

Reply:
ubuntu version removed, java version left as is